setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='fakenewsredis',
-    version='0.9.4',
+    version='0.9.5',
```

Comment:
Don't update the version yet; I'll do it when I make a new release.

Reply:
Done.